### PR TITLE
Add a common base type of IDE and diagnostics VS performance tests. 

### PR DESCRIPTION
**Pull request by mavasani**
This change adds a common base type for IDE and diagnostics performance tests.
Required for #15677, #15681 and #15682

### CODE REVIEW

File: src/Test/Perf/Utilities/VsPerfTest.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.IO;
+using static Roslyn.Test.Performance.Utilities.TestUtilities;
+
+namespace Roslyn.Test.Performance.Utilities
```
Comment (mavasani) [inline src/Test/Perf/Utilities/VsPerfTest.cs]: Code refactored from https://github.com/dotnet/roslyn-internal/pull/1841

**Comment by mavasani** [comment]
Tagging @dotnet/roslyn-perf for review

**Comment by mavasani** [comment]
@dotnet-bot retest windows_debug_vs-integration_prtest  please

**Comment by mavasani** [comment]
@dotnet-bot retest ubuntu_14_debug_prtest please

**Comment by mavasani** [comment]
Ping.. can someone take a look? This is just a test refactoring..

**Comment by mavasani** [comment]
Merging this in as it is a test only change, that doesn't need ask mode approval.

**Comment by rchande** [comment]
Missed the GH notification @mavasani. Belated 👍

**Pull request merged**